src/graphql-to-commits.ts

```diff
@@ -76,7 +90,7 @@ export async function graphqlToCommits(
   response: CommitHistoryGraphQLResponse
 ): Promise<CommitsResponse> {
   const commitHistory: CommitHistory =
-    response.repository.defaultBranchRef.target.history;
+    response.repository.refs.edges[0].node.target.history;
```

Comment:
I always end up feeling like I'm holding graphql wrong, if we gradually move more things to the v5 API, we might want to pull in an ORM.

Reply:
Yeah, the way we're doing this honestly feels a bit hacky to me, but I'd have to look at how others are using the API for an example of how to do it right. 😄 I was under the impression that there hasn't been a huge uptake on the graphql API for GitHub. We can talk about this thought later though.